docs/cri/config.md

```diff
@@ -503,11 +503,8 @@ runtime will be used. For example, see
 
 ## CNI Config Template
 
-Ideally the cni config should be placed by system admin or cni daemon like calico,
-weaveworks etc. However, there are still users using [kubenet](https://kubernetes.io/docs/concepts/cluster-administration/network-plugins/#kubenet)
-today, who don't have a cni daemonset in production. The cni config template is
-a temporary backward-compatible solution for them. This is expected to be
-deprecated when kubenet is deprecated.
+Ideally the cni config should be placed by system admin or cni daemon like calico, weaveworks etc.
+However, this is useful for the cases when there is no cni daemonset to place cni config.
```

Comment:
This part has to be fixed as well:
https://github.com/containerd/containerd/pull/8608/files#diff-91d0a4c61f6d3523b5a19717d1b40b5fffd7e392d8fe22aed7c905fe195b8902L405

Reply:
Why do you think this part should be removed?  I think it's a reasonable discussion; containerd generating the CNI config seems like a bit of an anti-pattern and is really just being done because it's convenient to do so.

Reply:
Because "This is a temporary" and "This will be deprecated" conflict with this PR.
I agree that it is a bit of an anti-pattern, and we do not encourage using it.

Reply:
@AkihiroSuda those lines are not there 
generated view  https://github.com/containerd/containerd/blob/cc03248ae0276a255e3db366b5016e5cc619c580/docs/cri/config.md#cni-config-template

Reply:
> @AkihiroSuda those lines are not there generated view https://github.com/containerd/containerd/blob/cc03248ae0276a255e3db366b5016e5cc619c580/docs/cri/config.md#cni-config-template

Still rendered in the "Full configuration" section.
(You have to click "Details")

Reply:
@AkihiroSuda sorry i missed it, thanks for pointing updated